src/resources/basis-worker.js

```diff
@@ -186,11 +219,8 @@ function BasisWorker() {
             // select output format based on supported formats
             basisFormat = hasAlpha ? alphaMapping[format] : opaqueMapping[format];
 
-            // transcode to uncompressed format if the texture is PVRTC or webgl1, and has invalid dimensions
-            const isPVRTC = (basisFormat === BASIS_FORMAT.cTFPVRTC1_4_RGB || basisFormat === BASIS_FORMAT.cTFPVRTC1_4_RGBA);
-            const notPOT = ((width & (width - 1)) !== 0) || ((height & (height - 1)) !== 0);
-            const notSquare = (width !== height);
-            if ((isPVRTC && (notPOT || notSquare)) || (!options.deviceDetails.webgl2 && notPOT)) {
+            // if image dimensions don't work on target target, fall back to uncompressed
```

Comment:
"target target"? I'm guessing you have one target too many? 😄

Reply:
thx :)